test/data/images-ref/centos-9-x86_64-qcow2.ref.yaml

```diff
@@ -0,0 +1,221 @@
+# generated via the image repo
+#
+# $ go build ./cmd/build/ && OSBUILD_TESTING_RNG_SEED=0 ./cmd/build/build -config config.json -distro centos-9 -type qcow2
+#
+# and stripping out the "inputs" and "curl" pieces
+version: "2"
+pipelines:
+  - name: build
+    runner: org.osbuild.centos9
+    stages:
+      - type: org.osbuild.rpm
+      - type: org.osbuild.selinux
+        options:
+          file_contexts: etc/selinux/targeted/contexts/files/file_contexts
+          labels:
+            /usr/bin/cp: system_u:object_r:install_exec_t:s0
+  - name: os
+    build: name:build
+    stages:
+      - type: org.osbuild.kernel-cmdline
+        options:
+          root_fs_uuid: d8bb61b8-81cf-4c85-937b-69439a23dc5e
```

Comment:
are we already that far with `otk` to generate this number in a variable (dynamically) and use it below?

…mainly to demonstrate this functionality - this might be too far from the current issue, so you can as well ignore this comment